core/camel-core/src/test/java/org/apache/camel/impl/ScheduledPollConsumerBackoffTest.java

```diff
@@ -22,10 +22,12 @@
 import org.apache.camel.spi.PollingConsumerPollStrategy;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.DisabledIfSystemProperty;
+import org.junit.jupiter.api.parallel.Isolated;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @DisabledIfSystemProperty(named = "ci.env.name", matches = "github.com", disabledReason = "Flaky on Github CI")
```

Comment:
Can it be removed?

Reply:
I don't think so, but it won't hurt to test. Let's give it a try.